test/runtime/Policies.go

```diff
@@ -951,4 +952,135 @@ var _ = Describe("RuntimeValidatedPolicyImportTests", func() {
 		Expect(strings.TrimSpace(policyMapsInVM.GetStdOut())).To(Equal(expected), "Only %s PolicyMap should be present", expected)
 
 	})
+
+	It("checks policy trace output", func() {
+
+		httpd2Label := "id.httpd2"
+		httpd1Label := "id.httpd1"
+		allowedVerdict := "Final verdict: ALLOWED"
+
+		By("Checking policy trace by labels")
+
+		By(fmt.Sprintf("Importing policy that allows ingress to %s from the host and %s", httpd1Label, httpd2Label))
+
+		allowHttpd1IngressHostHttpd2 := fmt.Sprintf(`
+			[{
+    			"endpointSelector": {"matchLabels":{"id.httpd1":""}},
+    			"ingress": [{
+        			"fromEndpoints": [
+            			{"matchLabels":{"reserved:host":""}},
+            			{"matchLabels":{"id.httpd2":""}}
+					]
+    			}]
+			}]`)
+
+		err := helpers.RenderTemplateToFile(policyJSON, allowHttpd1IngressHostHttpd2, 0777)
+		defer os.Remove(policyJSON)
+		Expect(err).Should(BeNil())
+
+		path := helpers.GetFilePath(policyJSON)
+		_, err = vm.PolicyImportAndWait(path, helpers.HelperTimeout)
```

Comment:
I will use `vm.PolicyRenderAndImport` instead of all Render ;-)

Reply:
Done.